components/WithdrawLiquidity/WithdrawLiquidityView.tsx

```diff
@@ -0,0 +1,108 @@
+        dropdownArrow: "w-12 h-8",
+        downIcon: "flex justify-center",
+        buttonDiv: "flex justify-center",
+        amountDiv: "border-2 border-gray-800 my-2 shadow-lg p-4 rounded-2xl text-white text-2xl font-semibold",
+        eachAmount: "flex justify-between p-2",
+        totalPriceDiv: "flex justify-between px-6 py-4 text-white text-2xl",
+        priceTag: "",
+        totalPrice: "text-right"
+      }
+
+        return (
+            <div className={style.wrapper}>
+              <div className={style.content}>
+                <div className={style.formHeader}>
+                  <div className={style.withdrawHeading}>Withdraw Liquidity</div>
+                </div>
+                <div>
+                  <div className={style.formLabel}>
+                    <p>
+                      Amount
+                    </p>
+                    <p>
+                      Detailed
+                    </p>
+                  </div>
+                  <div className={style.sliderStyle}>
+                    <p className={style.sliderPercent}>{sliderPercentage}%</p>
+                    <Slider
+                      value={sliderPercentage}
+                      onChange={changeSliderPercentage}
+                      onDone={onDone}
+                      thumbColor="rgb(51, 81, 70)"
+                      shadowColor="rgb(31, 41, 55)"
+                      sliderPathColor="rgb(31, 41, 70)"
+                    />
+                    <div className={style.percentageBox}>
+                      <p className={style.percentages} onClick={() => {changeSliderPercentage(25)}}>25%</p>
+                      <p className={style.percentages} onClick={() => {changeSliderPercentage(50)}}>50%</p>
+                      <p className={style.percentages} onClick={() => {changeSliderPercentage(75)}}>75%</p>
+                      <p className={style.percentages} onClick={() => {changeSliderPercentage(100)}}>Max</p>
+                    </div>
+                  </div>
+                  <div className={style.downIcon}>
+                    <ArrowDownIcon className={style.dropdownArrow} style={{color: 'white'}} />
+                  </div>
+
+                  <div className={style.amountDiv}>
+                    <div className={style.eachAmount}>
+                      <p>50</p>
+                      <p>ETH</p>
+                    </div>
+                    <div className={style.eachAmount}>
+                      <p>50</p>
+                      <p>ORBS</p>
+                    </div>
+                  </div>
+
+                  <div className={style.totalPriceDiv}>
+                    <div className={style.priceTag}>
+                      <p>Price</p>
+                    </div>
+                    <div className={style.totalPrice}>
+                      <p>1 ETH = 4295.45 ORBS</p>
+                      <p>1 ORBS = 0.000232804 ETH</p>
+                    </div>
+                  </div>
```

Comment:
Try to make the `amountDiv` styling to be more emphasized. 
The `totalPriceDiv` should be less emphasized and more compact.
Refer to Uniswap's `WithdrawLiquidity`:

<img width="496" alt="Screen Shot 2022-07-04 at 4 23 00 AM" src="https://user-images.githubusercontent.com/63836051/177144965-f7c09978-6008-4902-b19d-c297acb461e3.png">